### ISSUE TITLE
Were pre-trained models pre-trained only with unsupervised objectives?

### ISSUE THREAD

**Issue by marton-avrios** [google-research/text-to-text-transfer-transformer#314]
...or also on multiple different supervised tasks in a multi-task learning fashion? So are prefixes like "translate:" and "summarize:" necessary (or at least useful) if I want to fine-tune one of the downloadable pre-trained models for german to english translation or summarization?

**Comment by craffel** [comment]
The T5 checkpoints were trained on a multi-task mixture of supervised tasks and an unsupervised span-filling task. See section 3.7 of the paper for a description. The prefixes are not strictly necessary, but if you want to take full advantage (in terms of how fast the model converges) of the supervised tasks used during pre-training you should use the same prefixes.
